ini_settings.sh

```diff
@@ -374,14 +375,20 @@ function installDEBS () {
 			echo "Extracting ${ARCHIVE_FILES}"
 			ORIGINAL_DIR="$(pwd)"
 			cd "${TEMP_PATH}"
-			rm data.tar.xz > /dev/null 2>&1
-			ar -x "${TEMP_PATH}/${MAX_DEB_NAME}" data.tar.xz
+			rm data.tar.xz data.tar.gz > /dev/null 2>&1	
+			ar -x "${TEMP_PATH}/${MAX_DEB_NAME}" data.tar.*
 			cd "${ORIGINAL_DIR}"
 			rm "${TEMP_PATH}/${MAX_DEB_NAME}"
 			mkdir -p "${DEST_DIR}"
-			[ ! -f "${TEMP_PATH}/data.tar.xz" ] && echo "Error: no ${TEMP_PATH}/data.tar.xz found." && exit 1
-			tar -xJf "${TEMP_PATH}/data.tar.xz" --wildcards --no-anchored --strip-components="${STRIP_COMPONENTS}" -C "${DEST_DIR}" "${ARCHIVE_FILES}"
-			rm "${TEMP_PATH}/data.tar.xz" > /dev/null 2>&1
+			if [ -f "${TEMP_PATH}/data.tar.xz" ]
+			then
+				tar -xJf "${TEMP_PATH}/data.tar.xz" --wildcards --no-anchored --strip-components="${STRIP_COMPONENTS}" -C "${DEST_DIR}" "${ARCHIVE_FILES}"
```

Comment:
Hi, I am not sure you need to treat `gz` and `xz` differently there. `tar` can recognize the different compression and decide what decompression algorithm to use, so`tar xf data.tar.*` should work for both cases.

I would have to check if this is true for the `tar` version installed on the MiSTer though.